common/persistence/dataInterfaces.go

```diff
@@ -1521,6 +1532,7 @@ type (
 		DeleteWorkflowExecution(request *DeleteWorkflowExecutionRequest) error
 		DeleteCurrentWorkflowExecution(request *DeleteCurrentWorkflowExecutionRequest) error
 		GetCurrentExecution(request *GetCurrentExecutionRequest) (*GetCurrentExecutionResponse, error)
+		GetConcreteExecution(request *GetConcreteExecutionRequest) (*GetConcreteExecutionResponse, error)
```

Comment:
Could we reuse GetWorkflowExecution()?

Reply:
We could, but it selects and does lots of unnecessary stuff while we need to keep lookups as lite as possible for the scanner.